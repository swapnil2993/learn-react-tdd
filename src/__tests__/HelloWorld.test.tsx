import React from "react";
import { cleanup, render } from "@testing-library/react";
import HelloWorld from "../HelloWorld";

afterEach(cleanup);

test("test hello-world", () => {
  const { getByText } = render(<HelloWorld />);

  const h1 = getByText(/Hello World/i);
  expect(h1.textContent).toBe("Hello World");
  expect(h1).toMatchSnapshot();
});
