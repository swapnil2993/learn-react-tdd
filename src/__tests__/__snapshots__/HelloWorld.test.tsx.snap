// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test hello-world 1`] = `
<h1>
  Hello World
</h1>
`;
